### PR TITLE
Awesome books: plain JavaScript with objects

### PR DESCRIPTION
**Pull request by myaserkhan**
### Hi there 👋 

Untitled

In this particular project, @MYaserKhan and @Olivier-Kango we : 

- [ ] Create a collection that keeps a list of books.
- [ ] Create a function to add a new book to the collection, with title and author.
- [ ] Create a function to remove a book from the collection.
- [ ] Display all books saved in the collection in the top part of the page.
- [ ] When a user clicks the "Add" button: A new book is added to the collection.
- [ ] When a user clicks the "Remove" button:
    - [ ] The correct book is removed from the collection.
    - [ ] The correct book dissapears from the page.
  
NB: The data is also preserved in the browser's memory.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+// Local storage
+const bookForm = document.querySelector('#addBook');
+const content = JSON.parse(window.localStorage.getItem('storage'));
+function check() {
+  if (localStorage.getItem('storage')) {
+    bookForm.elements.title.value = content.title;
+    bookForm.elements.author.value = content.author;
+  }
+}
```
Comment (HAMMAS-SALEEM) [inline app.js]: - [ ] You guys have done a great job in add and delete operations :+1: but i can see that you didn't implement the localStorage properly. :fearful: Let me explain you what we are expecting you to do :+1:

1. First when you will add your book then it should be added in the localStorage and then a render function will run which will get the latest data from localStorage and will show it on the browser.

2. Remove function is a bit different. You need to get the localStorage first and store it into a variable. After storing it into a variable you need to filter that book from the variable which you want to remove. After filtering the book you need to set the filtered data means the filtered variable to the localStoroage then your render function will run and get the latest data of localStorage and will render it :thinking:

File: app.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+//   constructor (title, author) {
+//     this.title = title;
+//     this.author = author;
+//   }
+// }
+// const awesomeBooks = [];
+
+// awesomeBooks.push(new Book(title.value, author.value));
+
+// console.log(awesomeBooks);
```
Comment (HAMMAS-SALEEM) [inline app.js]: - [ ] Kindly remove this commented code because they are making your code not look clean :thinking:

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f574770523531743971623739534e464f74732f67697068792e676966

## Hey Team (@MYaserKhan, @Olivier-Kango) :wave:

## Highlights :1st_place_medal:

:heavy_check_mark: Following GitHub Flow
:heavy_check_mark: Descriptive PR Summary
:heavy_check_mark: Descriptive PR Title

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by RolandM99 — APPROVED** [review]
<h1>Status: Approve 🥇 ✔</h1>
Hi @MYaserKhan and @Olivier-Kango , 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**